app/views/govuk_publishing_components/components/docs/date_input.yml

```diff
@@ -0,0 +1,34 @@
+name: Form date input
+description: Use the date input component to help users enter a memorable date or one they can easily look up.
+accessibility_criteria: |
+  Inputs in the component must:
+
+  * accept focus
+  * be focusable with a keyboard
+  * be usable with a keyboard
+  * be usable with touch
+  * indicate when they have focus
+  * be recognisable as form input elements
+  * have correctly associated labels
+  * be of the appropriate type for their use, e.g. password inputs should be of type 'password'
```

Comment:
This looks like it was copied from somewhere else, can we be more specific? e.g. `be of the appropriate type for their use, in this case 'number'`

Reply:
Yep, copied from input component, will update with number as you stated above.